pygmt/src/histogram.py

```diff
@@ -0,0 +1,60 @@
+"""
+Histogram - Create a histogram
+"""
+from pygmt.clib import Session
+from pygmt.helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+
+
+@fmt_docstring
+@use_alias(
+    A="horizontal",
+    B="frame",
+    C="cmap",
+    G="fill",
+    J="projection",
+    R="region",
+    T="series",
+    W="pen",
+    c="panel",
+    l="label",
+    p="perspective",
+)
+@kwargs_to_strings(R="sequence", T="sequence")
+def histogram(self, table, **kwargs):
```

Comment:
Rename the parameter to `data`, similar to other functions?
```suggestion
def histogram(self, data, **kwargs):
```

Reply:
Some functions use data, some use table, I'd actually prefer table since histogram isn't a geographical plot.

Reply:
For the sake of consistency with the GMT terminology, my vote is it for to remain `table`

Reply:
@seisman Do you have an opinion on this? Is it fine if I leave it as `table`?

Reply:
Would also prefer table instead of data here.